java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S3751_java.html

```diff
@@ -4,7 +4,7 @@
 <p>So marking a sensitive method <code>private</code> may seem like a good way to control how such code is called. Unfortunately, not all Spring
 frameworks ignore visibility in this way. For instance, if you’ve tried to control web access to your sensitive, <code>private</code>,
 <code>@RequestMapping</code> method by marking it <code>@Secured</code> …​ it will still be called, whether or not the user is authorized to access
-it. That’s because AOP proxies are not applied to private methods.</p>
```

Comment:
This should not be reverted, the related PR is not yet merged! Can you review it?https://github.com/SonarSource/rspec/pull/584

Reply:
Done in https://github.com/SonarSource/sonar-java/pull/3845/commits/1c83f122fb865f783c90c543c5d5c48ad1b2da89.